src/Transaction/Request/Request.php

```diff
@@ -95,7 +95,7 @@ public function __call($method, $args)
 
         if ($prefix === 'set')
         {
-            return $this->offsetSet($param, $arg);
+            $this->offsetSet($param, $arg);
```

Comment:
Change this back or have the error in an else block otherwise it will always throw an error

Reply:
```php
$prefix = substr($method, 0, 3);
$param = substr($method, 3);
$arg = isset($args[0]) ? $args[0] : null;

if ($prefix === 'set')
{
    return $this->offsetSet($param, $arg);
} 

if ($prefix === 'get')
{
   return $this->offsetGet($param);
}
```

I'm not quite sure what this function is for because it's not being used.